cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -308,4 +310,3 @@ $Null = $ArtifactDownloadUrl, $NoModifyPath, $Help
 # Make Write-Information statements be visible
 $InformationPreference = "Continue"
 Install-Binary "$Args"
```

Comment:
```suggestion

# The default interactive handler
try {
  Install-Binary "$Args"
} catch {
  Write-Host $_
}
```

Reply:
thing I need to investigate: does this make the script think it succeeded (probably), and if so, what's the least noisy way to fix that.

Reply:
add `exit 1` to the end of the catch to set status properly

Reply:
Does powershell have the same exit status concept? Can we just `exit 1` or do something similar?